LORIS_indepth.html

```diff
@@ -670,8 +402,213 @@ <h2>LORIS - Github <font color="yellow"> - Open Source</font></h2>
                         <h2>LORIS - Github<font color="yellow"> Usage</font></h2>
                         <img src="images/github_usage2017.png">
```

Comment:
could you find/update this from our github? would be great to show through 2019

Reply:
good call, done